### PR TITLE
Jormun: removal of destineo and default scenario 

### PR DESCRIPTION
**Pull request by kinnou02**
This is a great day: destineo is officialy closed and we can start to
delete all of this specific code.
:tada: 

Before merging we need to:
  - [x] wait a little bit to be safe
  - [x] remove destineo tests from artemis

### CODE REVIEW

File: source/tyr/tyr/resources.py [hunk 1]
```diff
@@ -274,7 +274,7 @@ def put(self, id=None, name=None):
             type=str,
             case_sensitive=False,
             help='the name of the scenario used by jormungandr',
-            choices=['default', 'keolis', 'destineo', 'new_default', 'distributed'],
+            choices=['keolis', 'new_default', 'distributed'],
```
Comment (TeXitoi) [inline source/tyr/tyr/resources.py]: keolis?

Comment (kinnou02) [reply source/tyr/tyr/resources.py]: it is born dead: deleted

**Review by xlqian — COMMENTED** [review]
:+1:

**Review by xlqian — APPROVED** [review]

**Review by kadhikari — APPROVED** [review]

File: source/jormungandr/jormungandr/interfaces/v1/serializer/journey.py [hunk 2]
```diff
@@ -337,6 +337,7 @@ class JourneySerializer(PbNestedSerializer):
     calendars = CalendarSerializer(many=True, display_none=False)
     sections = SectionSerializer(many=True, display_none=False)
     debug = jsonschema.MethodField(schema_type=JourneyDebugSerializer(), display_none=False)
+    links = base.DescribedField(schema_type=LinkSchema(many=True))
```
Comment (pbougue) [inline source/jormungandr/jormungandr/interfaces/v1/serializer/journey.py:340]: :+1: Well seen, didn't think of same_journey_schedules

**Review by pbougue — APPROVED** [review]
:mushroom: 
:sweat_smile: The big thrill to remove such a big amount of python code...

**Review by pbougue — COMMENTED** [review]
I vote for a merge just after next release (so friday afternoon)

**Pull request merged**